src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -247,6 +247,10 @@
     <Compile Include="Microsoft\Data\SqlClient\EnclaveProviderBase.cs" />
     <Compile Include="Microsoft\Data\SqlClient\EnclaveSessionCache.cs" />
 	</ItemGroup>
+  <!-- netcoreapp 3.1 & netstandard 2.1 and above -->
+  <ItemGroup Condition="'$(OSGroup)' != 'AnyOS' AND '$(TargetFramework)' != 'netcoreapp2.1' AND '$(TargetFramework)' != 'netstandard2.0'">
+    <Compile Include="Microsoft\Data\SqlClient\SqlClientEventSource.NetCoreApp3.cs" />
```

Comment:
Since you have added condition for this file, there is no need to add suffix number '3' in its name. Starting from netcoreapp 3.1 and netstandard 2.1, this can be supported and extended in the future.

Reply:
I believe the file name will distinct the aim of this type for a developer just by seeing the file name beside the other files before getting into it.

Reply:
For now, netcoreapp 3.1 and netstandard 2.1 are the versions we support. It can be extended to support .net core 5.0 or other .net standard versions. By that time, the file name could be a bit misleading. This is just a suggestion anyway.